### PR TITLE
feature/dim_dates

### PR DESCRIPTION
**Pull request by suelen-nfr**
Addition of a dimension of dates for this package and corrections on the dbt project and readme

### CODE REVIEW

**Review by ArturWagner — APPROVED** [review]

File: models/staging/databricks_workflow_sources/stg_task_instance_databricks_workflow.sql [hunk 1]
```diff
@@ -2,26 +2,26 @@ with tasks as (
...
 renamed as (
     select 
-        {{ cast_as_string('task.task_key') }} as task_id
-        , {{ cast_as_string('job_id') }} as dag_id
-        , {{ cast_as_string('run_id') }} as run_id
-        , from_unixtime( task.start_time / 1000) as execution_date
-        , from_unixtime( task.start_time / 1000) as execution_start_date
-        , from_unixtime( task.end_time / 1000) as execution_end_date
+        cast(tasks.key as string) as task_id
+        , cast(job_id as string) as dag_id
```
Comment (PerriLucas) [inline models/staging/databricks_workflow_sources/stg_task_instance_databricks_workflow.sql:11]: in order to maintan a standard, it's necessary to add task. to this line and the one below

File: models/staging/databricks_workflow_sources/stg_task_instance_databricks_workflow.sql [hunk 2]
```diff
@@ -2,26 +2,26 @@ with tasks as (
...
+        , cast(run_id as string) as run_id
+        , to_timestamp( tasks.start_time / 1000) as execution_date
+        , to_timestamp( tasks.start_time / 1000) as execution_start_date
+        , to_timestamp( tasks.end_time / 1000) as execution_end_date
         , (execution_duration / 1000) as duration
```
Comment (PerriLucas) [inline models/staging/databricks_workflow_sources/stg_task_instance_databricks_workflow.sql:16]: line wasnt updated, but should also be adequated to the standard

File: README.md [hunk 3]
```diff
@@ -32,67 +34,34 @@ packages:
...
-    enabled_sources: ['airflow', 'adf'] # Possible values airflow, adf and databricks_workflow.
-    dbt_dag_monitoring:
-       dag_monitoring_start_date: cast('2023-01-01' as date) # edit date here
-```
-
-It's also necessary to switch on/off the models from dags sources you have data from.
-
-Example for running for adf and airflow:
+### Models:
+The functioning of the package on the desired platform depends on the configuration of dbt_project.yml. To define which platform we are transforming the data to, “true” information must appear in the “enabled” field of the referring platform. Other platforms must keep the field “enabled” filled with the information “false” .
```
Comment (PerriLucas) [inline README.md:38]: Nitpick:
Change:
 `To define which platform we are transforming the data to, “true” information must appear in the “enabled” field of the referring platform. Other platforms must keep the field “enabled” filled with the information “false” .`
To:
`To define which platform we are transforming the data to, the enabled field must be "true", for the desired platform, and "false" for all others.`

**Review by PerriLucas — CHANGES_REQUESTED** [review]
Small nitpick and codestyle

**Pull request merged**